shoop/core/settings.py

```diff
@@ -136,3 +136,6 @@
 
 #: The submission URL for Shoop's telemetry (statistics) system
 SHOOP_TELEMETRY_URL = "https://telemetry.shoop.io/collect/"
+
+#: Defines whether the default prices for products are assumed to be taxful or taxless.
+SHOOP_DEFAULT_PRICES_INCLUDE_TAX = False
```

Comment:
This should not be needed, since now you're adding the `Shop.prices_include_tax`.

Don't add this setting.